### PR TITLE
Add property for whether a device is attached

### PR DESCRIPTION
**Pull request by dnfield**
To enable https://flutter-review.googlesource.com/c/recipes/+/14400

### CODE REVIEW

File: config/devicelab_config.star [hunk 1]
```diff
@@ -392,6 +392,7 @@ def devicelab_prod_config(branch, version, ref):
                 "task_name": task,
                 "upload_metrics": branch in UPLOAD_METRICS_CHANNELS,
                 "use_cas": True,
+                "attached_device": True,
```
Comment (godofredoc) [inline config/devicelab_config.star:395]: Can we use device_os instead? if device_os is none then it does not have any device attached.

Comment (dnfield) [reply config/devicelab_config.star:395]: We can't access that from the recipe without making a network request to swarming.

Comment (dnfield) [reply config/devicelab_config.star:395]: (Tagged you in a relevant LUCI-users thread)

Comment (godofredoc) [reply config/devicelab_config.star:395]: Thanks!

**Review by godofredoc — APPROVED** [review]

**Pull request merged**